gemfile.d/test.rb

```diff
@@ -0,0 +1,7 @@
+group :test do
+  gem "minitest-tags"
+  gem "vcr", "< 4.0.0"
+  gem "rubocop-checkstyle_formatter"
```

Comment:
AFAIK we don't actually need this anymore. This used to be needed for Jenkins, but nowadays it can parse the normal progress format. Also, `theforeman-rubocop` still pulls it in.

Reply:
Fun fact: we don't even use `katello:rubocop:jenkins` today and don't even parse the results. In https://github.com/theforeman/jenkins-jobs/pull/400 I switch over to properly parsing the results.

Dropping it in https://github.com/Katello/katello/pull/10838